src/libs/DateUtils.ts

```diff
@@ -723,6 +723,30 @@ function formatWithUTCTimeZone(datetime: string, dateFormat: string = CONST.DATE
     return '';
 }
 
+/**
+ * Format the date from UTC to local timezone
+ * @param dateString
+ * @param dateFormat
+ * @returns
+ */
+
+function formatUTCToLocal(dateString: string, dateFormat = 'yyyy-MM-dd HH:mm:ss') {
```

Comment:
kindly re-use dateformat constants we have? If we don't, kindly create a constant for the same.